index.bs

```diff
@@ -189,6 +201,15 @@ Note: This metadata *appends* [=client hints token=]s to the [=environment setti
      
 Issue(110): Clarify detection of <{link}>, <{style}>, or <{script}> element execution.
 
+<dfn>Interaction with `ACCEPT_CH` Frame</dfn> {#interaction-with-accept-ch-frame}
```

Comment:
I don't think we want this to be a `<dfn>`, since it's just a heading and not a useful concept or algorithm. Instead, we can just make this a heading, and then created a hyperlink to the ID:

```
Interaction with `ACCEPT_CH` frame {#interaction-with-accept-ch-frame}
------------
```